src/app/pages/create-wallet/create-wallet.tsx

```diff
@@ -1,7 +1,158 @@
-import { Component, h } from 'preact';
+import { h, Component } from 'preact';
+import { route } from 'preact-router';
+import Button from 'preact-material-components/Button';
+import LayoutGrid from 'preact-material-components/LayoutGrid';
+import 'preact-material-components/Card/style.css';
```

Comment:
import of material components css should not be done at component level, also we don't use css we use scss. move this import to `src/app/styles/material-components.scss`